charts/vaultwarden/templates/ingress.yaml

```diff
@@ -1,3 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: customer-headers-configmap
+data:
+  Request-Id: $req_id
```

Comment:
Does this functionality hold harmless users that do not require this functionality, or should it also be placed behind an `{{- if $ingress.nginxIngressAnnotations }}` flag?

Reply:
Since this just defines a config map that is never referenced anywhere if the nginxIngressAnnotations variable is disabled, my guess would be that we wouldn't really have to hide it behind the flag.

That said, I'm more or less just getting started with kubernetes, so my guess might be off here. If I do add the flag, would I have to hide the separator (---) behind the flag as well?

Reply:
`ConfigMap` definitions should be added to `charts/vaultwarden/templates/configmap.yaml`.

Also, there is a typo in the name (should be "custom-headers-configmap not "customer-headers-configmap").